### DISCUSSION TITLE
MiniDexed with Pimoroni Pirate Audio Headphone

### DISCUSSION THREAD

**Discussion by postamesta** [probonopd/MiniDexed#741] [General]
Hi, i'm trying to use Minidexed with Pimoroni Pirate Audio Headphone (lcd + 4 buttons as page/up/down and confirm all in one).
Is there anyone who is trying this project? I'm not a programmer. I activated the audio DAC changing this lines to the mixidexed.ini file:

SoundDevice=i2s

Added some lines to config.txt to activate the soundboard:

dtoverlay=hifiberry-dac
gpio=25=op,dh

The audio works well, the sound is great. I'm now trying to install the repository st7789 for the screen, but nothing happens.
Some help here? Thank you

**Comment by probonopd** [comment]
Check this:

https://github.com/probonopd/MiniDexed/pull/652#issue-2280752338

**Reply by postamesta** [reply]
You simply blew away two months of attempts. Everything works FINE.
Thank you very much. A portable, compact DX7 with screen and navigation buttons.
The Pimoroni Pirate is a gift.

**Reply by probonopd** [reply]
> You simply blew away two months of attempts.

@diyelectromusic did 👍

**Comment by probonopd** [comment]
Nice video about the Pimoroni Pirate Audio:
https://www.youtube.com/watch?v=vCARbd6Lj40

It's significantly more expensive than a combination of the DAC and OLED from our wiki plus four buttons, but it comes at the advantage that absolutely no soldering and no cabling required.

**Comment by dariolp77** [comment]
Hi, I have the same setup with Raspberry 3 and Pimoroni audio module and I get no sound.

File Minidexed.ini edited as shown in tutorial but no sound. Screen is functional and shows menu but no sound from the jack when press keys from my Roland A49 midi keyboard. This is an screenshot of boot sequence 

![IMG_0652](https://github.com/user-attachments/assets/a2e35cf6-c19b-4fcc-b260-ef5b653e7bbe)

Any idea? Thanks

**Reply by diyelectromusic** [reply]
I'm pretty sure there is an "audio enable" GPIO pin that you have to preset to "on" in the config.txt file... but I thought I'd mentioned that somewhere, but I'm struggling to find a reference now...

Edit: it's mentioned in the first post here - doh :)

More on the product page: https://shop.pimoroni.com/products/pirate-audio-line-out?variant=31189750546515

Kevin

**Reply by dariolp77** [reply]
Nope. Added these 2 lines to my config.txt and still no sound...looks like my Roland A series keyboard is not supported...
![imagen](https://github.com/user-attachments/assets/59a3c58e-9434-43d5-98be-bdb2c4ce887c)

**Reply by dariolp77** [reply]
Solved! It was a problem from my keyboard mode. Reset to default and change driver mode recognition and everything was fine. Seems like Roland and Linux OS needs some work.

**Reply by probonopd** [reply]
Glad you got it working. Can you describe exactly what you changed, so that others can find it? Thanks.

MiniDexed is not using Linux.

**Reply by dariolp77** [reply]
In my Roland A49 you can change some settings with combo Function key and other key. I attached hdmi cable to a monitor to see if changes work and eureka! When press Function key and ADV key my Roland is detected.

**Reply by probonopd** [reply]
Thanks for letting us know.
